packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -490,8 +496,8 @@ export async function activate(context: vscode.ExtensionContext) {
   const orgList = new OrgList();
   orgList.displayDefaultUsername(defaultUsernameorAlias);
   context.subscriptions.push(registerOrgPickerCommands(orgList));
-  // await setupOrgBrowser(context, defaultUsernameorAlias);
 
+  await setupOrgBrowser(context, defaultUsernameorAlias);
```

Comment:
I think we should keep this commented out since we're always passing `false` for the tree view context in the following line.

Reply:
Ah yes, was meant to be commented still. Thanks